src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,19 +7,23 @@
 import edu.wpi.first.epilogue.Logged;
 import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.CommandGenericHID;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 import frc.robot.config.ControllerConfig;
 import frc.robot.subsystems.Algae;
+import frc.robot.subsystems.Coral;
 import frc.robot.subsystems.Drivetrain;
-import frc.robot.subsystems.Elevator;
 
 @Logged
 public class RobotContainer {
   private Drivetrain drivetrain = new Drivetrain();
   private Algae algae = new Algae();
-  private Elevator elevator = new Elevator();
+  private Coral coral = new Coral();
   private CommandXboxController controller = new CommandXboxController(0);
+  private CommandGenericHID totalController = new CommandGenericHID(0);
```

Comment:
This should be on port 1 instead of port 0 because the new gamepad is not the same as the xbox controller